docs/AboutUs.adoc

```diff
@@ -37,11 +37,11 @@ Responsibilities: Data
 '''
 
 === Tay Zi Hiang Willis
-image::m133225.jpg[width="150", align="left"]
-{empty}[http://github.com/m133225[github]] [<<johndoe#, portfolio>>]
+image::tzihiang.jpg[width="150", align="left"]
```

Comment:
Sorry I didn't notice this earlier, the website says that the file should be named as ".png" even if it's in jpg format.

Reply:
i think reuploading a png will be good. i tried with a jpg and it didn't work

Reply:
ok! i'll edit and push a new branch for that

Reply:
you can edit in your existing branch. after you are done editing, commit and then push it. the changes will reflect here